tuftool/src/root.rs

```diff
@@ -262,6 +272,21 @@ impl Command {
         println!("{}", key_id);
         write_file(path, &root)
     }
+
+    fn sign(path: &PathBuf, key_source: Box<dyn KeySource>) -> Result<()> {
+        let root: Signed<Root> = load_file(path)?;
+
+        let signed_root = SignedRole::new(
+            root.signed.clone(),
+            &root.signed,
+            &[key_source],
+            &SystemRandom::new(),
+        )
+        .context(error::SignRoot { path })?;
+
+        std::fs::write(&path, signed_root.buffer()).context(error::FileWrite { path: &path })?;
```

Comment:
Would be pretty funny to partially overwrite someone's root key with half of the signed data :D  Perhaps we should write to a temp file and move it over the original path?  (nit)

Reply:
Good point. :) I'll fix that.